### PR TITLE
Readme update :crocodile:

### PR DESCRIPTION
**Pull request by joesustaric**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -105,45 +108,46 @@ The following code will create a mock service on localhost:1234 which will respo
...
+  subject { AnimalServiceClient.new }
 
-  describe "get_something" do
+  describe "get_alligator" do
 
     before do
-      my_service_provider.given("something exists").
-        upon_receiving("a request for something").with(method: :get, path: '/something', query: '').
+      animal_service.given("alligator exists").
+        upon_receiving("a request for a alligator").
```
Comment (bethesque) [inline README.md]: Can we make this "an alligator exists"?

File: README.md [hunk 2]
```diff
@@ -153,35 +157,35 @@ end
 
 #### 5. Run the specs
 
-Running the consumer spec will generate a pact file in the configured pact dir (spec/pacts by default).
+Running the AnimalServiceClient spec will generate a pact file in the configured pact dir (spec/pacts by default).
 Logs will be output to the configured log dir that can be useful when diagnosing problems.
 
-Of course, the above specs will fail because the client method is not implemented, so next, implement your client methods.
+Of course, the above specs will fail because the Animal Service client method is not implemented, so next, implement your consumer client methods.
```
Comment (bethesque) [inline README.md]: Can we drop the word "consumer" here, it's not actually a consumer client, it's a provider client.

File: README.md [hunk 3]
```diff
@@ -226,7 +230,7 @@ Congratulations! You now have a failing spec to develop against.
 
 At this stage, you'll want to be able to run your specs one at a time while you implement each feature. At the bottom of the failed pact:verify output you will see the commands to rerun each failed interaction individually. A command to run just one interaction will look like this:
 
-    $ rake pact:verify PACT_DESCRIPTION="a request for something" PACT_PROVIDER_STATE="something exists"
+    $ rake pact:verify PACT_DESCRIPTION="a request for a alligator" PACT_PROVIDER_STATE="alligator exists"
```
Comment (bethesque) [inline README.md]: $ rake pact:verify PACT_DESCRIPTION="a request for an alligator" PACT_PROVIDER_STATE="an alligator exists"

**Comment by bethesque** [comment]
Looks good, just a few tweaks.

What do you think about drawing an arrow between the AnimalServiceClient and the AnimalService, to show that the AnimalServiceClient makes a request to the AnimalService?

**Comment by joesustaric** [comment]
Yup fixing and will resubmit. Good idea about the arrow too.

**Comment by bethesque** [comment]
You don't need to close this, just push the changes onto the same branch, and it will add them to the PR.

**Comment by joesustaric** [comment]
Woops sorry..

n000000b alert

**Comment by bethesque** [comment]
You'll be a pro by the end of hack day.

**Comment by bethesque** [comment]
One more tiny tweak! I'd like the diagram to convey the fact that the AnimalServiceClient is responsible for making the call to the Zoo App. What about drawing a line from the AnimalServiceClient itself to the Zoo App?

**Comment by joesustaric** [comment]
Yup that makes sense.
I both love and hate your attention to detail.

**Comment by bethesque** [comment]
Me too :P I do it just as much to myself!

**Comment by bethesque** [comment]
I'd accept it and do it myself, but I don't have the original.

**Comment by bethesque** [comment]
I hacked the image myself seeing as you guys had done so much work already! Thanks for your work, i think this is much better!

**Pull request merged**